app.js

```diff
@@ -93,6 +59,48 @@ router.post('/account', async ctx => {
     ctx.body = response;
 });
 
+const password = require('secure-random-password');
+const models = require('./models');
+const FROM_PHONE = '+14086179592';
```

Comment:
what is this phone number? comment please

Reply:
it is a a phone number from which codes are sent, thought this is more more or less obvious. Added env variable as well, going to document that in README

Reply:
README done